### PR TITLE
Implement firebase-config-ktx Kotlin library.

### PR DESCRIPTION
**Pull request by rlazo**

### CODE REVIEW

File: firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/TestConstructorUtil.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import com.google.firebase.remoteconfig.internal.ConfigCacheClient
+import com.google.firebase.remoteconfig.internal.ConfigFetchHandler
+import com.google.firebase.remoteconfig.internal.ConfigGetParameterHandler
+import com.google.firebase.remoteconfig.internal.ConfigMetadataClient
+import java.util.concurrent.Executor
+import com.google.firebase.abt.FirebaseABTesting
+
+// This method is a workaround for testing. It enable us to create a FirebaseRemoteConfig object
+// with mocks using the package-private constructor.
+fun FirebaseRemoteConfigBuilder(
```
Comment (vkryachko) [inline firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/TestConstructorUtil.kt]: nit: prefer method names that start with a lowerCase letter do distinguish it from types. Also `createRemoteConfig` might be a more suitable name for this

File: firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/ktx/RemoteConfigTests.kt [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+
+const val APP_ID = "APP_ID"
+const val API_KEY = "API_KEY"
+
+const val EXISTING_APP = "existing"
+
+open class DefaultFirebaseRemoteConfigValue : FirebaseRemoteConfigValue {
+    override fun asLong(): Long = throw UnsupportedOperationException("Unimplementend")
```
Comment (vkryachko) [inline firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/ktx/RemoteConfigTests.kt]: consider replacing throws with `TODO("Unimplemented")`

Comment (vkryachko) [reply firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/ktx/RemoteConfigTests.kt]: Also, what is the motivation for this class, can't `StringRemoteConfigValue` directly extend `FirebaseRemoteConfigValue`?

Comment (rlazo) [reply firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/ktx/RemoteConfigTests.kt]: Updated to use `TODO("Unimplemented")`.

The main reason to have these 2 classes clarity. Having the boilerplate in the `DefaultFirebaseRemoteConfigValue` makes `StringRemoteConfigValue` trivial to understand.

File: firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/TestConstructorUtil.kt (hunk 1 above)
Comment (rlazo) [reply firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/TestConstructorUtil.kt]: Done.

**Comment by rlazo** [comment]
@miraziz PTAL

**Pull request merged**